migrations/2022-02-24_224:44:31_add_kc_to_user.sql

```diff
@@ -0,0 +1,10 @@
+-- MDB generated migration file
+-- rambler up
+ALTER TABLE users
+    ADD COLUMN accounts_id VARCHAR(36) UNIQUE,
```

Comment:
is it must be not null? What will be default unique value?

Reply:
I guess you can't add this column in a single step.
You can add it with no constraints and then have another migration to introduce the non-null and uniqueness constraints.

Reply:
Between the two steps we should run some script to add `accounts_id` value for all existing users.
You can ask Amnon's help with extracting the data from keycloak.

Reply:
you mean that after script that will fill kc id I will add some hardcoded string as default?

Reply:
No. I mean after the script you can add the uniqueness constraint. From there on we enforce the kc ID for every user